robot/pmm/tests/browser/ServiceDeliveries/new_service_delivery.robot

```diff
@@ -18,67 +18,72 @@ Setup Test Data
     Click Object Button                    New
-    Load Page Object                       NewServiceDelivery                      ${ns}ServiceDelivery__c
-    verify current page title              New Service Delivery
-    Populate modal Form                    Service Delivery Name=${service_delivery_name}
-    ...                                    Service=&{service}[Name]
-    ...                                    Client=&{contact}[FirstName] &{contact}[LastName]
-    ...                                    Program Engagement=&{program_engagement}[Name]
+    Wait For Modal                         New                                     Service Delivery
+    Populate Modal Form                    Service Delivery Name=${service_delivery_name}
+    ...                                    Service=${service}[Name]
+    ...                                    Client=${contact}[FirstName] ${contact}[LastName]
+    ...                                    Program Engagement=${program_engagement}[Name]
     ...                                    Delivery Date=Today
     ...                                    Quantity=${quantity}
-    click Modal button                     Save
+    Click Modal Button                     Save
     Wait Until Modal Is Closed
-    verify page header                     Service Delivery
-    verify details                         Quantity                                contains             ${quantity}
-    verify details                         Service                                 contains             &{service}[Name]
-    verify details                         Program Engagement                      contains             &{program_engagement}[Name]
-    page should not contain                ${service_delivery_name}
-    verify page contains related list      Files
-    ${service_delivery_id} =               Save Current Record ID For Deletion     ${ns}ServiceDelivery__c
-    ${service_id} =                        Save Current Record ID For Deletion     ${ns}Service__c
-    ${program_engagement_id} =             Save Current Record ID For Deletion     ${ns}ProgramEngagement__c
-
+    Verify Page Header                     Service Delivery
+    Verify Details                         Quantity                                contains             ${quantity}
+    Verify Details                         Service                                 contains             ${service}[Name]
+    Verify Details                         Program Engagement                      contains             ${program_engagement}[Name]
+    Page Should Not Contain                ${service_delivery_name}
+    Verify Page Contains Related List      Files
+    Save Current Record ID For Deletion     ${ns}ServiceDelivery__c
 
 Create a Service Delivery via UI with Auto Name Override
     [Documentation]                        This test creates Service Delivery record and verifies that the Service Delivery record
```

Comment:
documentation for this test is exactly same as above one, please correct

Reply:
Fixed